pkg/injector/webhook.go

```diff
@@ -47,6 +49,9 @@ const (
 
 	// WebhookHealthPath is the HTTP path at which the health of the webhook can be queried
 	WebhookHealthPath = "/healthz"
+
+	// webhookTimeoutStr is the url variable name for timeout
```

Comment:
What does "url variable name for timeout" mean here?

Reply:
webhook urls come with a timeout variable in it, ie: `URL=/mutate?timeout=30s`

Reply:
Lets rename this to `webhookMutateTimeoutKey` or something along that line.